lib/lte_link_control/lte_lc.c

```diff
@@ -1260,19 +1260,21 @@ int lte_lc_system_mode_get(enum lte_lc_system_mode *mode,
 
 int lte_lc_func_mode_get(enum lte_lc_func_mode *mode)
 {
-	int err;
+	int err, mode_retval;
 
 	if (mode == NULL) {
 		return -EINVAL;
 	}
 
 	/* Exactly one parameter is expected to match. */
-	err = nrf_modem_at_scanf(AT_CFUN_READ, "+CFUN: %u", mode);
+	err = nrf_modem_at_scanf(AT_CFUN_READ, "+CFUN: %d", &mode_retval);
```

Comment:
The problem is that `enum lte_lc_func_mode` is 8 bits and `nrf_modem_at_scanf()` expects it to be 32 bits and aligned on a 4 byte boundary. I think this PR should fix all `nrf_modem_at_scanf()` calls where enums are used.

Instead of using a temporary variable, changing the conversion specifier to `%hhd` or `%hhu` (which indicates that the value is 8 bits) should also work.

Reply:
Thanks! Ill update the PR for all instances of this using the `%hhd` / `%hhu` specifier.